recipes/bioconductor-oligo/meta.yaml

```diff
@@ -0,0 +1,54 @@
+package:
+  name: bioconductor-oligo
+  version: "1.40.1"
+source:
+  fn: oligo_1.40.1.tar.gz
+  url: https://bioconductor.org/packages/3.5/bioc/src/contrib/oligo_1.40.1.tar.gz
+  md5: f5d262569456c869ea9c06d4afee0d53
+build:
+  script: R CMD INSTALL --build .
+  number: 0
+  skip: True # [osx]
+  rpaths:
+    - lib/R/lib/
+    - lib/
+requirements:
+  build:
+    - 'r-base >=3.2.2'
```

Comment:
It's not a good idea to pin it here. But I guess you need to to work around this bug?